include/p2gz/images.h

```diff
@@ -29,7 +29,7 @@ struct Image {
 struct ImageMgr {
 	ImageMgr();
 
-	~ImageMgr() { }
+	~ImageMgr() { delete &images; }
```

Comment:
This shouldn't dereference. 
```suggestion
	~ImageMgr() { delete images; }
```